build.gradle

```diff
@@ -0,0 +1,50 @@
+group 'compiler-fleet'
+version '1.0-SNAPSHOT'
+
+buildscript {
+    ext.kotlin_version = '1.2.+'
+    ext.argparser_version = "2.+"
+
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+    }
+}
+
+apply plugin: 'java'
+apply plugin: 'kotlin'
+apply plugin: 'application'
+
+repositories {
+    mavenCentral()
+    maven {
```

Comment:
I think xenomachina is available in `jcenter()` repository, no need to use its own one.

Reply:
Done